### PR TITLE
Ui Improvements and Updates to readme

### PR DESCRIPTION
**Pull request by YA010**
fixed css on mytab

added readme instructions to show how to run nextjs 

added a call to action box on homeui

added a image folder (no need to routinely link a non changing image better to keep it in 

this is after sign in 
after sign in

before sign in 
before sign in

let me know any parts i should remove / change or if just other ui devs will modify to fit styles

### CODE REVIEW

**Comment by pycui** [comment]
Thanks for the PR - I think we would need to align with our design so the UI change may need to hold a bit. The README and css fixes can probably go in first. Is it possible to split to a separate PR?

**Comment by YA010** [comment]
yep thats great , removed the component and the image and its folder

File: client/next-web/src/app/_components/MyTab.js [hunk 1]
```diff
@@ -28,13 +28,16 @@ export default function MyTab({ display }) {
...
-              <Avatar
+           
+            className="basis-48 p-2.5 bg-blue-500">
+               <Avatar
                 radius="sm"
                 src={character.image_url}
                 className="w-44 h-44"
               />
+            <CardFooter  className="p-0 flex
+    flex-col">
```
Comment (pycui) [inline client/next-web/src/app/_components/MyTab.js:39]: seems like unnecessary linebreak

File: client/next-web/src/app/_components/ExploreTab.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ export default function ExploreTab({ characters, display }) {
         characters.map(character => (
           <Card
             key={character.character_id}
-            isFooterBlurred
+
```
Comment (pycui) [inline client/next-web/src/app/_components/ExploreTab.js:17]: unnecessary newline?

File: README.md [hunk 3]
```diff
@@ -216,6 +216,13 @@ ELEVEN_LABS_API_KEY=<api key>
             npm start
             ```
             After running these commands, a local development server will start, and your default web browser will open a new tab/window pointing to this server (usually http://localhost:3000).
+        - **Option 3**: Running the client in Nextjs.
```
Comment (pycui) [inline README.md:219]:
```suggestion
        - **Option 3 (experimental)**: Running the client in Nextjs.
```

File: client/next-web/src/app/_components/MyTab.js (hunk 1 above)
Comment (pycui) [reply client/next-web/src/app/_components/MyTab.js:39]: still here?

Comment (YA010) [reply client/next-web/src/app/_components/MyTab.js]: oh yea forgot about mytab.js too  , only fixed exploretab.js but sent a commit now

**Pull request merged**